tests/frontend/test_search.py

```diff
@@ -152,6 +163,21 @@ def test_selected_result_is_highlighted(base_url, selenium, variables):
     assert page.search.highlighted_suggestion
 
 
+@pytest.mark.nondestructive
+def test_search_box_character_limit(base_url, selenium, variables):
+    page = Home(selenium, base_url).open()
+    term = ''
+    # put 100 characters into term
+    for i in range(10):
+        term += '0123456789'
+    search = page.search.search_for(term)
+    assert term in search.results_info.text.split('\"')[-2]
+    # make term have 101 characters
+    term += 'e'
+    search = page.search.search_for(term)
```

Comment:
at this point, we should also check that the search field is not accepting the extra `e` character, namely, the character limit is indeed 100.